src/commands/integration-test-go-test.yaml

```diff
@@ -22,7 +22,7 @@ steps:
       # TODO remove this part after switch to Go modules.
       name: "architect/integration-test-go-test: Move code to $GOPATH"
       command: |
-        mkdir -p $(go env GOPATH | cut -d ':' -f 1)/src/github.com/${CIRCLE_PROJECT_USERNAME} && mv ~/project $GOPATH/src/github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}
+        mkdir -p $(go env GOPATH | cut -d ':' -f 1)/src/github.com/${CIRCLE_PROJECT_USERNAME} && mv ~/project $(go env GOPATH | cut -d ':' -f 1)/src/github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}
```

Comment:
I missed updating the 2nd $GOPATH. Tested in https://github.com/giantswarm/app-operator/pull/212.